docs/src/@primer/gatsby-theme-doctocat/nav.yml

```
@@ -1,2 +1,6 @@
 - title: Colors
   url: /colors
+- title: Spacing & size
```

To follow our [content guidelines](https://primer.style/design/foundations/content#what-to-avoid) 😉:
> Use the word “and” instead of an ampersand or the “+” sign.

```suggestion
- title: Spacing and size
```